final release/model_sklearn.py

```diff
@@ -1,37 +1,52 @@
 """Creates various Sklearn models"""
+import inspect
```

Comment:
`inspect` doesn't seem to be used, so remove it

Reply:
ok